rbac/server/api/main.py

```diff
@@ -78,6 +91,11 @@ def load_config(app):
     app.config.DEBUG = bool(get_config("DEBUG"))
     app.config.LOGGING_LEVEL = get_config("LOGGING_LEVEL")
     app.config.SECRET_KEY = get_config("SECRET_KEY")
+    app.config.SWAGGER_UI_CONFIGURATION = {
```

Comment:
What is this Swagger UI configuration doing? I couldn't find any documentation on what exactly it means to set up these configs.